elasticsearch/src/main/java/org/hibernate/search/elasticsearch/logging/impl/Log.java

```diff
@@ -159,14 +159,13 @@ BulkRequestFailedException elasticsearchBulkRequestFailed(String request, String
 	SearchException facetingRequestHasUnsupportedType(String facetingRequestType);
 
 	@Message(id = ES_BACKEND_MESSAGES_START_ID + 27,
-			value = "The 'indexNullAs' property for field '%2$s' needs to represent a Boolean to match the field type of the index. "
-					+ "Please change value from '%1$s' to represent a Boolean." )
-	SearchException nullMarkerNeedsToRepresentABoolean(String proposedTokenValue, String fieldName);
+			value = "The 'indexNullAs' property for Boolean fields must represent a Boolean ('true' or 'false')." )
+	IllegalArgumentException invalidNullMarkerForBoolean();
```

Comment:
When it comes to these exceptions I think that quoting the value which the user attempted to use is helpful to understand the error. I see the previous version of the error had it, why remove this?